### PR TITLE
feat(app.js state) modified file structure and added ui state components

### PR DESCRIPTION
**Pull request by darrinmn9**

### CODE REVIEW

File: client/src/components/hero_order/heroOrderCtrl.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+(function() {
+  'use strict';
+
+  angular.module('app.hero_order', [])
+  .controller('HeroOrderCtrl', [ function(){
```
Comment (tatsumitagawa) [inline client/src/components/hero_order/heroOrderCtrl.js]: No space before `function` and space after ().

`[function() {`

File: client/src/components/choice/choiceCtrl.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+(function() {
+  'use strict';
+
+  angular.module('app.choice', [])
+  .controller('ChoiceCtrl', [ function(){
```
Comment (tats-tagawa) [inline client/src/components/choice/choiceCtrl.js]: No space before `function` and space after ().

`[function() {`

File: client/src/components/hero_location/heroLocationCtrl.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+(function() {
+  'use strict';
+
+  angular.module('app.hero_location', [])
+  .controller('heroLocationCtrl', [ function(){
```
Comment (tats-tagawa) [inline client/src/components/hero_location/heroLocationCtrl.js]: see ↑ . Same for rest of the controllers.

**Comment by tats-tagawa** [comment]
Everything's good except for two small styling errors. Please fix and commit. Also on Trello, move the card under Pending Pull Requests.

**Pull request merged**